AndroidSDKCore/src/main/java/com/leanplum/internal/VarCache.java

```diff
@@ -87,6 +87,7 @@
   private static boolean silent;
   private static int contentVersion;
   private static Map<String, Object> userAttributes;
+  private static Map<String, Object> variantDebugInfo;
```

Comment:
Similar comment elsewhere, but this should be its own class, not just a map.

Reply:
^